helm/ph-ee-engine/values.yaml

```diff
@@ -455,6 +455,8 @@ importer_es:
   requests:
     memory: "512M"
     cpu: "100m"
+  reporting:
```

Comment:
Its because of github indentation
<img width="995" alt="Screenshot 2022-06-09 at 2 26 50 PM" src="https://user-images.githubusercontent.com/92036361/172807828-0ce9c8c0-53de-4707-a03c-7896537490e0.png">